src/IIIFPresentation/API/Attributes/EtagCachingAttribute.cs

```diff
@@ -47,7 +47,7 @@ ResultExecutionDelegate next
 
             responseHeaders.CacheControl = new CacheControlHeaderValue() // how long clients should cache the response
             {
-                Public = !requestHeaders.Headers.Keys.Contains("IIIF-CS-Show-Extra"),
+                Public = !requestHeaders.Headers.Keys.Contains("X-IIIF-CS-Show-Extras"),
```

Comment:
Should this use something similar to `ShowExtraProperties()` extension method, so that it's all contained in 1 place?